sentinel/src/config.js

```diff
@@ -42,7 +42,11 @@ const initFeed = () => {
     .on('change', change => {
       if (change.id === 'settings') {
         logger.info('Reloading configuration');
-        initConfig();
+        initConfig().then(() => {
+          if (config['reschedule']) {
+            require('./schedule').checkSchedule();
```

Comment:
Had to quickly devise a way of restarting the scheduler on the fly for e2e tests. Stefan has a PR that adds support to stop/restart api or sentinel in e2e, so once that is merged I plan to remove this and restart sentinel when I need it to re-schedule.